src/bridge/cockpitdisksamples.c

```diff
@@ -135,9 +135,15 @@ cockpit_disk_samples (CockpitSamples *samples)
           && g_ascii_isdigit (dev_name[strlen (dev_name) - 1]))
         continue;
 
+      // ignore nvme partitions
+      if (g_str_has_prefix (dev_name, "nvme") && g_strrstr (dev_name, "p"))
+        continue;
+
```

Comment:
also somewhat unrelated, separate commit would be probably nicer

Reply:
Agreed. This also needs some justification -- why do you want to do that? I quite like seeing my partitions on the metrics page..
```
/dev/nvme0n1p3 on /sysroot type ext4 (rw,relatime,seclabel)
/dev/nvme0n1p3 on / type ext4 (rw,relatime,seclabel)
/dev/nvme0n1p3 on /usr type ext4 (ro,relatime,seclabel)
/dev/nvme0n1p3 on /var type ext4 (rw,relatime,seclabel)
/dev/nvme0n1p2 on /boot type ext4 (rw,relatime,seclabel)
/dev/nvme0n1p4 on /var/home type ext4 (rw,relatime,seclabel)
/dev/nvme0n1p1 on /boot/efi type vfat (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=ascii,shortname=winnt,errors=remount-ro)
```